tests/OperationProcessor.spec.ts

```diff
@@ -224,4 +224,27 @@ describe('OperationProcessor', async () => {
     const didDocument = await operationProcessor.resolve(didUniqueSuffix);
     expect(didDocument).toBeUndefined();
   });
+
+  it('should return undefined for deleted did', async () => {
+    const numberOfUpdates = 1;
+    const ops = await createUpdateSequence(didUniqueSuffix, createOp!, cas, numberOfUpdates, privateKey);
+
+    for (let i = 0 ; i < ops.length ; ++i) {
+      await operationProcessor.processBatch([ops[i]]);
+    }
```

Comment:
Sanity: why wouldn't the code use `processBatch(ops)` directly?

Reply:
No reason. Fixed it (and also the source this was cut-n-pasted from).